### ISSUE TITLE
Run pulseaudio-dlna as root

### ISSUE THREAD

**Issue by Lahorde** [masmu/pulseaudio-dlna#137]
I cannot get it working. It is blocked at " Checking pulseaudio processes ...". It works fine when called by non root user.

```
sudo pulseaudio-dlna --debug
12-13 22:48:59 pulseaudio_dlna.__main__                       INFO     Running as root. Starting daemon ...
12-13 22:48:59 pulseaudio_dlna.daemon                         INFO     Checking pulseaudio processes ...
```

So I continue calling it as described here : 
https://github.com/masmu/pulseaudio-dlna/issues/52#issuecomment-164300821

**Comment by masmu** [comment]
Which version are you using?

**Comment by masmu** [comment]
If you are still using 0.4.6 try upgrading to 0.4.7. There was an problem with the _psutil_ library which is fixed in 0.4.7.

**Comment by Lahorde** [comment]
I just tried it successfully with 0.4.7.
Thanks
